arch/arc/include/arc-connect-priv.h

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
filename with dash instead of underscore?

Reply:
this file doesn't exist in updated version.